sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -53,6 +53,15 @@ now raise more concrete exception other than `MessageSettleFailed` and `ServiceB
   - Added support for Python 3.9.
   - Fixed bug where amqp message `footer` and `delivery_annotation` were not encoded into the outgoing payload.
 
+**Breaking Changes**
+
+* `amqp_message` has been renamed to `amqp_annotated_message` for cross-sdk consistency.
+* All `name` parameters in `ServiceBusAdministrationClient` are now precisely specified ala `queue_name` or `rule_name`
+* `via_partition_key` is no longer exposed, this is pending a full implementation of transactions as it has no external use.  If needed, the underlying value can still be accessed in `Message.amqp_annotated_message.annotations`.
+* `Message.properties` has been renamed to `Message.application_properties` for consistency with service verbiage.
+* Sub-client (`ServiceBusSender` and `ServiceBusReceiver`) `from_connection_string` initializers have been made internal until needed.  Clients should be initialized from root `ServiceBusClient`.
+* `Message.label` has been renamed to `Message.subject`.
+
```

Comment:
we already have *Breaking changes" section above, we shall merge them into one.

Reply:
Those had separated so far over time I didn't even notice we had two in the merge, nice catch XD